app/src/main/java/space/taran/arknavigator/mvp/model/repo/scores/ScoreStorage.kt

```diff
@@ -0,0 +1,17 @@
+package space.taran.arknavigator.mvp.model.repo.scores
+
+import space.taran.arknavigator.mvp.model.repo.index.ResourceId
+import space.taran.arknavigator.utils.Score
+
+interface ScoreStorage {
+
+    fun contains(id: ResourceId): Boolean
+
+    fun setScore(id: ResourceId, score: Score)
+
+    fun getScore(id: ResourceId): Score?
+
```

Comment:
I think you can return non-nullable type since it can only be null in aggregated mode, but we have `contains` function for check

Reply:
Yeah, and I realise it always returns a value